test/wrappers/ortools_test.rb

```diff
@@ -5770,4 +5770,25 @@ def test_self_selection_should_not_change_vehicle_ids
       end
     end
   end
+
+  def test_quantity_precision
+    problem = VRP.basic
+    problem[:services].each{ |service|
+      service[:quantities] = [{ unit_id: 'kg', value: 1.001 }]
+    }
+    problem[:vehicles].each{ |vehicle|
+      vehicle[:capacities] = [{ unit_id: 'kg', limit: 3 }]
+    }
+
+    result = OptimizerWrapper.wrapper_vrp('demo', { services: { vrp: [:ortools] }}, TestHelper.create(problem), nil)
+    assert_equal 1, result[:unassigned].size, 'The result is expected to contain 1 unassigned'
```

Comment:
why are we failing the test if there is more than 1 unassinged. This makes the test "flaky".

All we know is that there should be at least one unassigned due to capacity constraint. 
```suggestion
    assert_operator  result[:unassigned].size, :>=, 1, 'The result is expected to contain at least 1 unassigned due to capacity'
```

Reply:
If we have more than one unassigned, we have probably applied a multiplier on the quantities and not on the capacities.

Reply:
I think we should have a proper unit_test (instead of a wrapper_vrp) for that -- for example "stub"ing the solve and checking if a value 1.003 becomes 1003 or not. This is not the right way, wrapper_vrp is a big black box and this test adds more false negatives than protection.

Reply:
not solve, run_solver..